### PR TITLE
Upgrade decidim version to 0.26

### PR DESCRIPTION
**Pull request by ClaudiaMoPr**
This PR upgrades the application to Decidim version 0.26.
Release notes: https://github.com/decidim/decidim/releases/tag/v0.26.0

On deploy it will require to execute the following commands:

For locked users are present in global search, to update the search and make them disappear, Run in a bin/rails console or create a migration with:
`Decidim::User.find_each(&:try_update_index_for_search_resource)`

For fixing statistic in comments. There's a rake task that you need to run:
`bin/rails decidim_comments:update_participatory_process_in_comments`

For Base64 images migration run the following task:
`bin/rails decidim:active_storage_migrations:migrate_inline_images_to_active_storage[admin_email]`

### CODE REVIEW

File: .slug-post-clean [hunk 1]
```diff
@@ -1,6 +1,5 @@
-node_modules
```
Comment (tramuntanal) [inline .slug-post-clean:1]: This line should not be removed
```
node_modules/
```

File: config/secrets.yml [hunk 2]
```diff
@@ -11,8 +11,12 @@
...
 
 default: &default
-  aws_access_key_id: <%= ENV["AWS_ACCESS_KEY_ID"] %>
-  aws_secret_access_key: <%= ENV["AWS_SECRET_ACCESS_KEY"] %>
+  aws:
+    access_key_id: <%= ENV["AWS_S3_ACCESS_KEY_ID"] %>
+    region: <%= ENV["AWS_S3_REGION"] %>
+    bucket: <%= ENV["AWS_S3_BUCKET"] %>
+    secret_access_key: <%= ENV["AWS_S3_ACCESS_KEY"] %>
+
```
Comment (tramuntanal) [inline config/secrets.yml:19]: this change should be also updated at `config/storage.yml` please

**Review by tramuntanal — CHANGES_REQUESTED** [review]
I've just found a couple of things to consider @LauraJaimeC

File: config/secrets.yml (hunk 2 above)
Comment (laurajaime) [reply config/secrets.yml:19]: 😅 I restore it, I did not realize that change...

Comment (tramuntanal) [reply config/secrets.yml:19]: No... it comes from the clean app. Better get the modification from the clean_app and update `config/storage.yml` and `config/initializers/carrierwave.rb` accordingly. Like in Calafell.

File: config/storage.yml [hunk 3]
```diff
@@ -8,10 +8,10 @@ local:
 
 amazon:
   service: S3
-  access_key_id: <%= Rails.application.secrets.aws_access_key_id %>
-  secret_access_key: <%= Rails.application.secrets.aws_secret_access_key %>
-  region: <%= ENV.fetch("AWS_REGION", "eu-west-3") %>
-  bucket: <%= ENV.fetch("AWS_BUCKET_NAME", "decidim-aytopinto-prod") %>
+  access_key_id: <%= Rails.application.secrets.aws.access_key_id %>
```
Comment (tramuntanal) [inline config/storage.yml]: `Rails.application.secrets.aws` returns a hash and we can not access the keys of a hash with `.access_key_id` for example.

Better use `Rails.application.secrets.aws["access_key_id"]

**Review by tramuntanal — CHANGES_REQUESTED** [review]
All AWS keys should be accessed with standard hash notation.
See the failing test suite

**Review by tramuntanal — APPROVED** [review]

**Pull request merged**